wremnants/theory_tools.py

```diff
@@ -190,25 +200,23 @@ def define_pdf_columns(df, dataset, pdfs, noAltUnc):
         entries = 1 if i != 0 and noAltUnc else pdfInfo["entries"]
         start = 0 if "first_entry" not in pdfInfo else pdfInfo["first_entry"]
 
-        df = df.Define(tensorName, f"auto res = wrem::clip_tensor(wrem::vec_to_tensor_t<double, {entries}>({pdfBranch}, {start}), 10.); res = nominal_weight/nominal_pdf_cen*res; return res;")
+        if i == 0:
+            df = df.Define(tensorName, f"auto res = wrem::clip_tensor(wrem::vec_to_tensor_t<double, {entries}>({pdfBranch}, {start}), 10.); res = nominal_weight*res; return res;")
+            df = df.Define("nominal_pdf_cen", f"{tensorName}(0)")
+            df = df.Redefine("nominal_weight", "nominal_weight*nominal_pdf_cen")
```

Comment:
It looks to me that ```nominal_weight = nominal_weight^2 * pdf_weight```, so the nominal weight gets squared?

Reply:
Aaah, good catch, yes! Will fix